src/components/LogRow/BaseRow/Highlighter.tsx

```diff
@@ -0,0 +1,34 @@
+import { memo, useMemo } from "react";
+import { highlightHtml } from "utils/renderHtml";
+
+interface HighlighterProps {
+  ["data-cy"]?: string;
+  children: string;
+  color?: string;
+  highlights?: RegExp;
+  searchTerm?: RegExp;
+}
+
+const Highlighter: React.FC<HighlighterProps> = memo((props) => {
```

Comment:
Can you write some unit tests for this as well?

Reply:
I moved some tests out of `Row.test.tsx` into these other files because it was getting really redundant (i.e. if `Highligher` is testing these things, and `Row` contains `Highlighter`, then `Row` probably doesn't need to test it again)